config/software/chef.rb

```diff
@@ -119,6 +86,7 @@
   appbundle 'ohai'
 
   # Clean up
+  # Move this to a more appropriate place that's common to all software we ship.
```

Comment:
Is this a FIXME? It's not clear if you mean to move these files or move this block of code.